addons/im_livechat/static/src/public/main.js

```diff
@@ -20,3 +21,13 @@ const serviceRegistry = registry.category('services');
 serviceRegistry.add('messaging', messagingService);
 serviceRegistry.add('messagingValues', messagingValuesService);
 serviceRegistry.add('messaging_service_to_legacy_env', makeMessagingToLegacyEnv(owl.Component.env));
+
+if (isAvailable) {
+    const button = new LivechatButton(
+        rootWidget,
+        serverUrl,
+        options,
+    );
+    button.appendTo(document.body);
+    window.livechat_button = button;
```

Comment:
This does not look used

Reply:
It's removed in a follow-up commit (the one introducing the `PublicLivechatService`).

I try to avoid adapting in-between commits, as I don't intend to keep this split in commits. I'll squash most things anyway.